packages/d3fc-webgl/test/util/vertextShaderUtil.js

```diff
@@ -0,0 +1,52 @@
+const Compiler = require('glsl-transpiler');
+
+export const getShaders = element => {
+    let _program;
+    try {
+        const bar = element.decorate(program => {
+            _program = program;
+            throw new Error();
+        });
+        bar();
+    } catch (e) {
+        // no-op
```

Comment:
Might be worth throwing a known error message i.e. `new Error('HORRIBLE HACK')` and asserting that's the received error i.e. `catch (e) { if (e.message !== 'HORRIBLE HACK') { throw e; } ... }`

Reply:
Good idea.